thread/src/components/Common/TransparentButton.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Remove unused variables, and all the extra styling which is not needed. Also `full` prop makes the button take the whole space, so therefore centers the text
```
import React from 'react';
import { Text, Button } from 'native-base';

const TransparentButton = ({ label, onPress }) => (
  <Button transparent light full onPress={onPress}>
         <Text>{label}</Text>
  </Button>
);

export default TransparentButton;
```